micromamba/tests/test_list.py

```diff
@@ -110,11 +118,17 @@ def test_list_subcommands(
             else:
                 assert "#" not in output
     else:
-        if canonical_flag == "--canonical":
+        if canonical_flag in ["-c", "--canonical"]:
             items = ["conda-forge/", "::"]
             for output in outputs_list:
                 assert all(i in output for i in items)
                 assert " " not in output
+        else:
+            if export_flag in ["-e", "--export"]:
+                items = ["conda-forge/", "::", " "]
+                for output in outputs_list:
+                    # assert all(i not in output for i in items)
+                    assert output.count("=") == 2
```

Comment:
This is slightly more stricter.
```suggestion
                    assert len(output.split("=")) == 3
```